<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlgCreateConfigFile</class>
 <widget class="QDialog" name="DlgCreateConfigFile">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>612</width>
    <height>499</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Crear fichero de configuraci√≥n</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Plantilla</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="2">
    <widget class="QComboBox" name="templateComboBox"/>
   </item>
   <item row="1" column="0" colspan="3">
    <widget class="QTableWidget" name="valuesTable">
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
     <column>
      <property name="text">
       <string>Nombre</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Valor</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QPushButton" name="choosePathButton">
     <property name="text">
      <string>Escoger...</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Fichero final</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLineEdit" name="outFileLineEdit"/>
   </item>
   <item row="3" column="2">
    <widget class="QPushButton" name="createButton">
     <property name="text">
      <string>Crear</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="outPathLineEdit"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Carpeta</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>templateComboBox</tabstop>
  <tabstop>valuesTable</tabstop>
  <tabstop>outPathLineEdit</tabstop>
  <tabstop>choosePathButton</tabstop>
  <tabstop>outFileLineEdit</tabstop>
  <tabstop>createButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
